controllers/windowsmachine_controller.go

```diff
@@ -247,6 +247,12 @@ func (r *WindowsMachineReconciler) Reconcile(ctx context.Context, request ctrl.R
 		err := r.client.Get(ctx, kubeTypes.NamespacedName{Namespace: machine.Status.NodeRef.Namespace,
 			Name: machine.Status.NodeRef.Name}, node)
 		if err != nil {
+			// Delete a configured/running machine if no associated node can be determined (node object deleted)
+			if k8sapierrors.IsNotFound(err) {
+				log.Info("machine has no associated node", "name", machine.GetName())
```

Comment:
Shouldn't log `Error`,  given you are returning a `err != nil`  ?

Reply:
I don't think log.Error is required here as we are trying to remediate the node not being found by deleting the machine.